R/ide.R

```diff
@@ -9,6 +9,20 @@
 # the URL may be specified with or without the protocol and port; this function
 # will try both http and https and follow any redirects given by the server.
 validateServerUrl <- function(url, certificate = NULL) {
+
+  valid <- validateConnectUrl(url, certificate)
```

Comment:
Could we document that this function has a side-effect of registering a server if it discovers that the input URL can be transformed into a valid server?